util/chplenv/chpl_jemalloc.py

```diff
@@ -22,6 +29,17 @@ def get():
     if mem_val != 'jemalloc' and jemalloc_val != 'none':
       error("CHPL_JEMALLOC must be 'none' when CHPL_MEM is not jemalloc")
 
+    if flag == 'host':
+      platform_val = chpl_platform.get('target')
+      linux = platform_val.startswith('linux')
+      darwin = platform_val.startswith('darwin')
+
+      if linux and mem_val == 'jemalloc' and jemalloc_val == 'system':
+        error("CHPL_JEMALLOC=system is not supported on Linux for host builds")
```

Comment:
This is probably captured in an issue, but what prevents us from using a system jemalloc on linux (I don't actually think this is an important use-case, more just curious)

Reply:
That was my call because different distros package it so differently, some provide `jemalloc-config`, some have the `.pc` file, fedora has a `.so.n` with no `.so`.